packages/ui-extensions-server-kit/src/components/Status/Status.scss

```diff
@@ -0,0 +1,19 @@
+.Status {
+  font-size: 13px;
+  border-radius: 10px;
```

Comment:
Can you add border-radius into the global styling similar to how we have it here: `packages/dev-console-app/src/theme.module.css`